src/lib/openapi/spec/api-token-schema.ts

```diff
@@ -0,0 +1,50 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { ApiTokenType } from '../../types/models/api-token';
+
+export const apiTokenSchema = {
+    $id: '#/components/schemas/apiTokenSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['username', 'type'],
+    properties: {
+        secret: {
+            type: 'string',
+        },
+        username: {
+            type: 'string',
+        },
+        type: {
+            enum: Object.values(ApiTokenType),
```

Comment:
Should probably stick to `string` here as well, if we're supposed to support any casing.